addons/website_event_meet/static/src/js/website_event_create_meeting_room_button.js

```diff
@@ -45,7 +41,10 @@ publicWidget.registry.websiteEventCreateMeetingRoom = publicWidget.Widget.extend
      * @override
      */
     destroy: function () {
-        $('.o_wevent_create_meeting_room_modal').remove();
+        const modal = document.querySelector(".o_wevent_create_meeting_room_modal");
+        if (modal) {
+            modal.parentNode.removeChild(modal);
+        }
```

Comment:
@bso-odoo I believe this could also be simply `modalEl.remove()` right ?

Reply:
Oh right :grimacing:

Reply:
Also, `createModelEl` was the right name: it is an element. (It also felt weird when first reading it though)

Reply:
@bso-odoo So are we moving ahead with `modelEl` or should i change it to `createModalEl` ?

Reply:
This one is `modalEl`, but the one on top had a better name before last change, I'll put a message there to be more clear.